run/types/DeviceWrapper.ts

```diff
@@ -89,7 +88,8 @@ export class DeviceWrapper {
   }
 
   public async getCssProperty(name: string, elementId: string): Promise<string> {
-    return this.toShared().getCssProperty(name, elementId);
+    const cssProp: string = await this.toShared().getCssProperty(name, elementId);
+    return cssProp;
```

Comment:
not sure why this was needed

Reply:
I think this may have been a solution for checking avatar picture change and that the colour was correct. Not using it so will remove